src/swagger/api.yaml

```diff
@@ -8602,14 +8602,11 @@ definitions:
         type: string
       type:
         enum:
-          - TEST
-          - TEST2
-          - WEBHOOK
-          - OAUTH2_TOKEN_TRANSFORM
-          - SAML_ASSERTION_TRANSFORM
-          - PRE_REGISTRATION
-          - IMPORT_TRANSFORM
-          - PASSWORD_IMPORT
+          - com.okta.oauth2.tokens.transform
```

Comment:
Will update the Open API spec also with this.

Reply:
https://github.com/okta/openapi/pull/246/files

Reply:
Cool, does this `oauth2.tokens.transform` automatically get translated to this `OAUTH2_TOKENS_TRANSFORM `
(if so that is great!)

Reply:
<img width="710" alt="image" src="https://user-images.githubusercontent.com/61501885/85053740-d6a6a680-b14f-11ea-99a7-e71c40cd29d4.png">